### PR TITLE
CmpTabnineHub command

### PR DESCRIPTION
**Pull request by bilucodota**
Hi @tzachar 

Thanks for maintaining this plugin! 

Please check out this PR - I've added a command to open Tabnine Hub - as this is requested by many of our users.
I'm not a lua developer nor a cmp user. So good chance I've made it completely wrong. 

Feel free to change anything if needed.

Thanks!

### CODE REVIEW

**Comment by tzachar** [comment]
can you please apply the following patch on top of your PR?
```lua
diff --git a/lua/cmp_tabnine/init.lua b/lua/cmp_tabnine/init.lua
index f8e05d9..ef0b25d 100644
--- a/lua/cmp_tabnine/init.lua
+++ b/lua/cmp_tabnine/init.lua
@@ -5,9 +5,11 @@ local M = {}
 
 M.setup = function()
   vim.schedule(function()
-    tabnine_source = source.new()
+    local tabnine_source = source.new()
     cmp.register_source('cmp_tabnine', tabnine_source)
-    vim.cmd [[command! CmpTabnineHub lua tabnine_source.open_tabnine_hub()]]
+    vim.api.nvim_add_user_command('CmpTabnineHub', function()
+      tabnine_source:open_tabnine_hub()
+    end, { force = true })
   end)
 end
 
diff --git a/lua/cmp_tabnine/source.lua b/lua/cmp_tabnine/source.lua
index 301b82a..a16ab71 100644
--- a/lua/cmp_tabnine/source.lua
+++ b/lua/cmp_tabnine/source.lua
@@ -136,24 +136,17 @@ function Source.new()
   return self
 end
 
-
-Source.open_tabnine_hub = function()
+Source.open_tabnine_hub = function(self)
   local req = {}
   req.version = '3.3.0'
   req.request = {
     Configuration = {
-      quiet = false
+      quiet = false,
     },
   }
 
-  local bin = binary()
-  if not bin then
-    return
-  end
-
-  local hub = fn.jobstart({ bin, '--client=cmp.vim' })
-
-  pcall(fn.chansend, hub, fn.json_encode(req) .. '\n')
+  -- pcall(fn.chansend, hub, fn.json_encode(req) .. '\n')
+  pcall(fn.chansend, self.job, fn.json_encode(req) .. '\n')
 end
 
 Source.is_available = function()

```

**Comment by tzachar** [comment]
Also, can you add something in the README?

**Comment by bilucodota** [comment]
@tzachar done

File: README.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ tabnine:setup({
 })
 ```
 
+## Log in to your account
```
Comment (tzachar) [inline README.md:63]: Please change to something which does not sound like you *have* to log in to your account.

Comment (bilucodota) [reply README.md:63]: How does "Activate your pro account" sounds to you?

Comment (tzachar) [reply README.md:63]: I don't like the `pro` part....
Maybe `Configure Tabnine or Login to Your Account` ???

Comment (bilucodota) [reply README.md]: done

**Comment by tzachar** [comment]
Thank you for your patience :)

**Comment by marcinmaciukiewicz** [comment]
I'm using version of nvim 0.6.1 - and this line cause a nil exception:

vim.api.nvim_add_user_command('CmpTabnineHub',


the nvim_add_user_command doesn't exists at vim.api - is this my local issue ?

**Comment by tzachar** [comment]
This was introduced lately, in nvim 0.7.
I'll change it to be backwards compatible.

**Comment by tzachar** [comment]
@marcinmaciukiewicz 
should be solved now.

**Comment by marcinmaciukiewicz** [comment]
yep, it's working - I'm using last stable - since 0.7 had some performance issues

**Pull request merged**